src/System.Reflection/tests/EventInfoTests.cs

```diff
@@ -34,8 +34,16 @@ public void IsMulticast_ReturnsTrue(Type type, string name)
         public void GetAddMethod(Type type, string name, bool isVisible)
         {
             EventInfo eventInfo = GetEventInfo(type, name);
-            Assert.Equal(!isVisible, eventInfo.GetAddMethod() == null);
-            Assert.Equal(!isVisible, eventInfo.GetAddMethod(false) == null);
+            if (isVisible)
+            {
+                Assert.NotNull(eventInfo.GetAddMethod());
+                Assert.NotNull(eventInfo.GetAddMethod(false));
+            }
+            else
+            {
+                Assert.Null(eventInfo.GetAddMethod());
+                Assert.Null(eventInfo.GetAddMethod(false));
+            }
```

Comment:
In general I'm not seeing the value of expanding out these asserts like this.  I could see in this case inverting both sides to involve fewer logical negatives, e.g.
```C#
Assert.Equal(isVisible, eventInfo.GetAddMethod() != null);
Assert.Equal(isVisible, eventInfo.GetAddMethod(false) != null);
```
but I don't think we want to double the asserts like this in all such cases.